module/zfs/spa_misc.c

```diff
@@ -2169,6 +2169,7 @@ spa_import_progress_init(void)
 	    spa_import_progress_list;
 
 	procfs_list_install("zfs",
+	    "spa",
```

Comment:
Shouldn't this be NULL to avoid changing the existing path?

Reply:
Yeah, I think we can live without "spa" there. I just noticed @allanjude committed per-dataset kstats to FreeBSD, too, with the "zfs/POOL" style naming that's being eliminated here. We might want to avoid changing that now.

@allanjude thoughts?

Reply:
I'd prefer to see the / go away as well.

Reply:
I can modify the version I committed to freebsd 12 (the older illumos ZFS) to match this if we sort out what we want in the next few days, so it can be merged in time for 12.2

Reply:
I think the changes in __kstat_create are all you would need for that. It's simply splitting the string at '/' and adding another node if necessary. A bit of a hack, but meh.

Reply:
My major concern here is to avoid breaking anything currently using these paths.  So for the moment let's make sure they don't change.  We can definitely consider changing the layout after this is merged.

Reply:
@behlendorf the paths are antithetical to sysctl naming convention, so perhaps keep them on Linux and split them elsewhere.